tutorials/2. Advanced PyDEns.ipynb

```diff
@@ -0,0 +1,800 @@
+{
```

Comment:
<p>Almost everywhere in this notebook the plot is done manually. Is this intended?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/pydens/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>